shared/lib/site-settings/exporter/reducers.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+import Immutable from 'immutable';
+import debugModule from 'debug';
+
+/**
+ * Internal dependencies
+ */
+import {
+	TOGGLE_EXPORTER_ADVANCED_SETTINGS,
+	TOGGLE_EXPORTER_SECTION
+} from '../action-types';
+
+const debug = debugModule( 'calypso:exporter' );
+
+const initialUIState = Immutable.fromJS( {
+	advancedSettings: {
+		isVisible: false,
+		posts: {
+			isEnabled: true
+		},
+		pages: {
+			isEnabled: true
+		},
+		feedback: {
+			isEnabled: true
+		}
+	}
+} );
+
+/**
+ * Reducer for managing the exporter UI
+ *
+ * @param  {Object} state  Current state
+ * @param  {Object} action Action payload
+ * @return {Object}        Updated state
+ */
+export function ui( state = initialUIState, action ) {
```

Comment:
Hmm, I'm not sure if that would be necessary until this file grows. This reducer is still related to the exporter and there are no further levels of reducer nesting. Here I've basically followed the pattern introduced in [/shared/lib/sharing/publicize/reducers.js](https://github.com/Automattic/wp-calypso/blob/master/shared/lib/sharing/publicize/reducers.js)